server/__tests__/unit/services/Subscriptions.js

```diff
@@ -1,50 +1,78 @@
+const mongoose = require("mongoose");
 const {
   createSubscription,
   fetchSubscriptionByuserid,
 } = require("../../../services");
-
-const { mockUsers } = require("../../../utils/mocks/Users");
-const { SubscriptionCollection } = require("../../../utils/firestore");
+const { mockSubscriptions } = require("../../../utils/mocks/Subscriptions");
 const { Subscriptions } = require("../../../models");
+const { MongoMemoryServer } = require("mongodb-memory-server");
+
+beforeAll(async () => {
```

Comment:
Is this required here ? Some test files doesn't have `afterAll` and `beforeAll`.